python/reg_interface/scripts/repeated_reg_read.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+
+from reg_utils.reg_interface.common.reg_xml_parser import getNode, parseXML
+from reg_utils.reg_interface.common.print_utils import printRed
+from reg_utils.reg_interface.common.jtag import initJtagRegAddrs
+from reg_utils.reg_interface.common.reg_base_ops import *
+
+import time
+import socket
+
+import argparse
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser(description='Arguments to supply to repeated_reg_read.py')
+
+    parser.add_argument('reg_name', metavar='reg_name', type=str,help='Name of the register to read')
+    parser.add_argument('nreads', metavar='nreads', type=str,help='Number of reads')
+    parser.add_argument('sleeptime', metavar='sleeptime', type=str,help='Amount of time to sleep in microseconds')
+    parser.add_argument('-f','--filename', metavar='filename', type=str, help="Filename to which output information is written", default="repeated_reg_read.txt")
+    parser.add_argument('--card', metavar='card', type=str,help='CTP7 hostname (has no effect if you are running on a hostname that starts with eagle)',default="eagle26")
+
```

Comment:
Why are `nreads` and `sleeptime` of type `str`?
In your usage you're casting them anyway, why not just let the option parser do that for you?